.semaphore/semaphore.yml

```diff
@@ -103,9 +107,14 @@ blocks:
               value: g++-8
           commands:
             - echo "Setting up build..."
-            - meson setup build.gcc -Dusewebcam=true -Dusemididrum=true -Dusewebserver=true -Dstoponwarning=true
+            - meson setup --buildtype release build.gcc -Dusewebcam=true -Dusemididrum=true -Dusewebserver=true -Dwarningaserror=true
+            - artifact push job --expire-in 1w build.gcc/meson-info
+            - artifact push job --expire-in 1w build.gcc/meson-logs
+            - artifact push job --expire-in 1w build.gcc/build.ninja
+            - artifact push job --expire-in 1w build.gcc/compile_commands.json
             - echo "Building..."
             - meson compile -C build.gcc
+            - artifact push job --expire-in 1w build.gcc/.ninja_log
```

Comment:
Like mentioned before, these artifacts aren't quite usefull for non-logged in users so i'd either find a way to print them to stdout or just remove them. Keeping them as an artifact is pretty useless imho.